lunch-go-where/src/utils/apiCalls.js

```diff
@@ -2,68 +2,88 @@ import axios from "axios";
 const baseURL = "http://localhost:5001";
 
 const login = async (body) => {
-	const response = await axios.post(baseURL + "/login", body, {
-		withCredentials: true
-	});
-	console.log(response);
+    const response = await axios.post(baseURL + "/login", body, {
+        withCredentials: true,
+    });
+    console.log(response);
 };
 
 const getExistingReview = async (id) => {
-	const response = await axios.get(baseURL + "/review/" + id, { withCredentials: true });
-	return response;
+    const response = await axios.get(baseURL + "/review/" + id, {
+        withCredentials: true,
+    });
+    return response;
 };
 
 const postNewStall = async (body) => {
-	const response = await axios.post(baseURL + "/new", body, {
-		withCredentials: true
-	});
-	console.log("POSTED NEW STALL", response);
-	return response;
+    const response = await axios.post(baseURL + "/new", body, {
+        withCredentials: true,
+    });
+    console.log("POSTED NEW STALL", response);
+    return response;
 };
 
 const postNewReview = async (body) => {
-	await axios.post(baseURL + "/review", body, { withCredentials: true });
+    await axios.post(baseURL + "/review", body, { withCredentials: true });
 };
 
 const updateReview = async (body) => {
-	await axios.patch(baseURL + "/review", body, { withCredentials: true });
+    await axios.patch(baseURL + "/review", body, { withCredentials: true });
 };
 
 const getRecommendedStalls = async () => {
-	const response = await axios.get(baseURL + "/home");
-	return response.data;
+    const response = await axios.get(baseURL + "/home");
+    return response.data;
 };
 
 const postSignUp = async (body) => {
-	const response = await axios.post(baseURL + "/signup", body, {
-		withCredentials: true
-	});
-	console.log("POSTED SIGNUP", response);
-	return response.data;
+    const response = await axios.post(baseURL + "/signup", body, {
+        withCredentials: true,
+    });
+    console.log("POSTED SIGNUP", response);
+    return response.data;
 };
 
 const checkAuthUser = async () => {
-	const response = await axios.get(baseURL, {
-		withCredentials: true
-	});
-	return response;
+    const response = await axios.get(baseURL, {
+        withCredentials: true,
+    });
+    return response;
 };
 
 const getOneStall = async (id) => {
-	console.log(id);
-	const response = await axios.get(baseURL + "/food/stall/" + id, { withCredentials: true });
-	return response;
+    console.log(id);
+    const response = await axios.get(baseURL + "/food/stall/" + id, {
```

Comment:
**No changes needed here,** just a second reminder that again, in your PR on the backend, you changed the API endpoint on the backend to `/food/:id`, but didn't change the route here! If we were using this endpoint on the front, the feature would break. :P